src/style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  margin: 0;
+  padding: 10px;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+main {
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 2px solid black;
```

Comment:
- [Optional] you added a border and your todo list cards are larger than the given design please update it to have a better design :+1: